### PR TITLE
fix(#39): Parse commands with arguments

### PR DESCRIPTION
**Pull request by elenavoronin**

### CODE REVIEW

File: kotishell/include/minilibft.h [hunk 1]
```diff
@@ -55,4 +55,9 @@ char		*mini_concat(const char *s1, const char *s2);
 */
 void		*mini_free(void *p);
 
+/**
+ * @brief joins 2 strings and returns the resulted string
```
Comment (Oktosha) [inline kotishell/include/minilibft.h:59]: Maybe add more useful info? That the strings must be null-terminated, not-null, and that the new string is allocated on heap and the old ones aren't freed and on malloc error a NULL is returned?

File: kotishell/src/minilibft/mini_strjoin.c [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+#include "minilibft.h"
+
+char	*mini_strjoin(char *s1, char *s2)
+{
+	char	*res;
+	int		i;
+	int		j;
+
+	if (s1 == NULL || s2 == NULL)
```
Comment (Oktosha) [inline kotishell/src/minilibft/mini_strjoin.c:21]: Maybe assert?

File: kotishell/src/minilibft/mini_strjoin.c [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+		return (NULL);
+	res = mini_malloc(sizeof(char) * (mini_strlen(s1) + mini_strlen(s2) + 1));
+	if (!res)
+		return (0);
```
Comment (Oktosha) [inline kotishell/src/minilibft/mini_strjoin.c:25]: Write 0 as NULL?

File: test/ep_tests/ep_main.c [hunk 4]
```diff
@@ -92,42 +92,43 @@ void	EP_test_expand(t_ks_kotistate *kotistate, t_lx_result *lx_res, t_EP_dummy_t
 
 int	main(void)
 {
```
Comment (Oktosha) [inline test/ep_tests/ep_main.c:94]: Uncomment?

File: test/ps_tests/Makefile [hunk 5]
```diff
@@ -25,7 +25,8 @@ minilibft/mini_malloc.c \
 minilibft/mini_putstr_fd.c \
 minilibft/mini_assert.c \
 minilibft/mini_substr.c \
-minilibft/mini_write.c
+minilibft/mini_write.c \
+minilibft/mini_strjoin.c
```
Comment (Oktosha) [inline test/ps_tests/Makefile:29]: Add test bacK?

**Review by Oktosha — CHANGES_REQUESTED** [review]

File: kotishell/include/minilibft.h (hunk 1 above)
Comment (elenavoronin) [reply kotishell/include/minilibft.h:59]: updated

File: kotishell/src/minilibft/mini_strjoin.c (hunk 3 above)
Comment (elenavoronin) [reply kotishell/src/minilibft/mini_strjoin.c:25]: okaaaay

File: kotishell/src/minilibft/mini_strjoin.c (hunk 2 above)
Comment (elenavoronin) [reply kotishell/src/minilibft/mini_strjoin.c:21]: yes, changed to assert

**Review by Oktosha — APPROVED** [review]

**Pull request merged**